R/annotation.R

```diff
@@ -2,24 +2,45 @@
 coding_vc = c("Frame_Shift_Del", "Frame_Shift_Ins", "In_Frame_Del", "In_Frame_Ins", "Missense_Mutation", "Nonsense_Mutation", "Nonstop_Mutation", "Splice_Region", "Splice_Site", "Targeted_Region", "Translation_Start_Site")
 
 
-#' Annotate and auto-drop a MAF data frame with existing blacklists to remove variants that would be dropped during the merge process.
+#' @title Annotate SSM with Blacklists
 #'
-#' @param mutations_df df with mutation data.
-#' @param seq_type The seq_type of your mutations if you prefer to apply only the corresponding blacklist (default is to use all available)
-#' @param unix_group The unix group (should be the same for all)
-#' @param tool_name The tool or pipeline that generated the files (should be the same for all)
-#' @param flavour Set to "clustered" if you want to use the blacklist for the new and improved SLMS-3 outputs (otherwise leave empty)
-#' @param genome_build The genome build projection for the variants you are working with (default is grch37)
-#' @param project_base Optional: A full path to the directory that your blacklist_file_pattern is relative to
-#' @param blacklist_file_pattern Optional: A string that contains the relative path to your blacklist file from after the project_base (i.e. results) with any wildcards surrounded with curly braces
-#' @param drop_threshold The minimum count from one of the blacklists to drop a variant
-#' @param verbose For debugging, print out a bunch of possibly useful information
-#' @param invert USE WITH CAUTION! This returns only the variants that would be dropped in the process (opposite of what you want, probably)
+#' @description Annotate and auto-drop a MAF data frame with existing blacklists.
```

Comment:
It's because I added punctuation at the end of every parameter description.

Reply:
Did you do it by hand or there are some scripts/functions? I think it might not be sustainable in a long-term future unless we find a way to consistently follow this. It might be that it is now consistently introduced everywhere but the new functions will still be missing these so will then end up with a need another review/PR and it will keep going on like this. Just thinking if we can automate the things like punctuation.

Reply:
My understanding is that Adam is now using a spellchecking add-on for VS Code. Maybe we should all try to use that if we're writing documentation?

Reply:
Yes! Let's try to adopt it! Adam can you advertise this extension on Slack so we all will use the same add on (and any settings that need to be modified from default)? Thanks!

Reply:
Yes, It's working out great. The extension I am using is called Spell Right. I'll advertise it on Slack!